### PR TITLE
javascript-carousel exercise - adrian arias

### PR DESCRIPTION
**Pull request by adrianm-arias**
carousel-challenge

### CODE REVIEW

File: javascript-carousel/main.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    $dotFour.className = 'fa-solid fa-circle';
+    $dotFive.className = 'fa-regular fa-circle';
+  } else if (counter === 4) {
+    $dotOne.className = 'fa-regular fa-circle';
+    $dotTwo.className = 'fa-regular fa-circle';
+    $dotThree.className = 'fa-regular fa-circle';
+    $dotFour.className = 'fa-regular fa-circle';
+    $dotFive.className = 'fa-solid fa-circle';
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-carousel/main.js:109]: When the user manually changes the slide, the countdown to the next auto-advance should start over:

---

image

Comment (TimelikeClosure) [inline javascript-carousel/main.js:109]: This is a lot of repetition. I suggest finding a way to slim this down by introducing a dynamic parameter.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by adrianm-arias** [comment]
carousel-timer-fixed

File: javascript-carousel/main.js (hunk 1 above)
Comment (adrianm-arias) [reply javascript-carousel/main.js:121]: sorry, just saw this.

**Review by Zhiwutian — APPROVED** [review]
Great Updates Adrian!

**Pull request merged**